integration/examples/jib-sync/build.gradle

```diff
@@ -13,11 +13,11 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 dependencies {
-  compile "org.springframework.boot:spring-boot-starter-web"
-  compile "org.springframework.boot:spring-boot-starter-actuator"
-  compile "org.springframework.boot:spring-boot-devtools"
+  implementation "org.springframework.boot:spring-boot-starter-web"
+  implementation "org.springframework.boot:spring-boot-starter-actuator"
+  implementation "org.springframework.boot:spring-boot-devtools"
```

Comment:
I see that you updated Maven `pom.xml` to selectively add the devtools dependency with a profile. Should we also apply this only when `-Psync` for Gradle?

Reply:
I thought I did, but I guess I didn't. :\